cra-README.md

```diff
@@ -1,2433 +0,0 @@
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
-
```

Comment:
Why did you remove this Readme? This seems pretty useful for future maintainers of the package.

Reply:
It is a boilerplate created with the project template. It can easily be found online in a much easier to digest format.

Reply:
Okay, I think it would be a good idea to put a [link to the upstream project](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md) in our project's readme. At least I don't see any references to the source of the original code anywhere else otherwise.